utility/data_fetcher.go

```diff
@@ -257,6 +258,19 @@ func (d *ManualDecoder) Headers(base map[string]interface{}) http.Header {
 	return httpHeader
 }
 
+// UserAgentHeader fetches all `user-agent` values from a given header and combines them into one string.
+// Values are separated by `;`.
+func (d *ManualDecoder) UserAgentHeader(header http.Header) string {
+	var ua string
```

Comment:
```suggestion
	return strings.Join(header[textproto.CanonicalMIMEHeaderKey("User-Agent")], "; ")
```

Reply:
rfc 2616/7230 say to use comma:

> A recipient MAY combine multiple header fields with the same field
   name into one "field-name: field-value" pair, without changing the
   semantics of the message, by appending each subsequent field value to
   the combined field value in order, separated by a comma.